04-vector-raster-integration-advanced.Rmd

```diff
@@ -159,29 +156,37 @@ dev.off()
 ```
 
 The graphic above illustrates the extent of several of the spatial layers that 
-we have worked with in this vector data lesson series including:
+we have worked with in this vector data lesson series:
 
-* Area of interest (AOI).
-* A roads/trails.
-* And plot locations.
-* A canopy height model (CHM) in GeoTIFF format.
+* Area of interest (AOI) -- blue
+* Roads and trails -- purple
+* Vegetation plot locations -- black
 
-However, sometimes we have a raster file that is must larger than our study 
-area or area of interest. It is often more efficient to `crop` the raster to the 
-extent of our study area, to keep file sizes smaller as we process our data. 
+and a raster file, that we will introduce this lesson: 
 
+* A canopy height model (CHM) in GeoTIFF format -- green
 
-## Import Data
-We will use four vector shapefiles with data on (field site boundary,
-roads/trails, tower location, and study plot locations) and one raster GeoTIFF
-file, a Canopy Height Model for the Harvard Forest, Massachusetts. 
+The raster files that we have to work with will rarely match the desired vector 
```

Comment:
Please remove this paragraph and replace with what was there. The point of this is not that we're trying to match things up. The biggest use cases for cropping are 1) the raster is big so it's more efficient 2) pretty mapping. 

Sometimes we have a raster file that is must larger than our study 
+and a raster file, that we will introduce this lesson: 
-area or area of interest. It is often more efficient to `crop` the raster to the 
-extent of our study area, to keep file sizes smaller as we process our data.